### PR TITLE
Added a resource on OpenAI Models Deployment

### PR DESCRIPTION
**Pull request by Odeyiany2**
Hii @Sammybams, I was implementing Azure OpenAI models in the scripts, and I came across a very important resource that wasn't included in the resource section. This will help in understanding how contributors can deploy the required models after creating the Azure OpenAI resource, to get the necessary environment variables.
Please look over it. Thank you.

### CODE REVIEW

**Review by Sammybams — APPROVED** [review]
Thank you very much @Odeyiany2.

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 API_KEY=
 ENDPOINT=
-OPENAI_API_VERSION=
+OPENAI_API_VERSION= #input the version of the openai
```
Comment (Sammybams) [inline .env.example]:
```suggestion
OPENAI_API_VERSION=
```

**Comment by Sammybams** [comment]
I have also removed your change to the .env.example file. The reason is in the commit message.

**Comment by Odeyiany2** [comment]
> I have also removed your change to the .env.example file. The reason is in the commit message.

Yes, thank you. That was a mistake.

**Pull request merged**